release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2101,6 +2144,45 @@ Workaround: Manually or by using a daemon set, modify the file `/etc/cni/tuning/
 
 * For {product-title} {product-version}, the Driver Toolkit (DTK) container image requires the `ubi9` image as the second layer of the software stack for building driver containers. If you attempt to use the `ubi8` image as the second layer in the your software stack, you will receive a build error. (link:https://issues.redhat.com/browse/OCPBUGS-11120[*OCPBUGS-11120*])
 
+[id="ocp-4-13-ran-known-issues"]
+* If you specify an invalid subscription channel in the subscription policy that you use to perform a cluster upgrade, the {cgu-operator-first} indicates that the upgrade is successful immediately after {cgu-operator} enforces the policy because the `Subscription` resource remains in the `AtLatestKnown` state.
+(link:https://issues.redhat.com/browse/OCPBUGS-9239[*OCPBUGS-9239*])
+
+* `kdump` fails to generate the `vmcore` crash dump file on HPE Edgeline e920t and HPE ProLiant DL110 Gen10 servers with Intel E810 NIC and ice driver installed.
+(link:https://issues.redhat.com/browse/RHELPLAN-138236[*RHELPLAN-138236*])
+
+* In {ztp}, when you provision a managed cluster that contains more than a single node using a `SiteConfig` CR, disk partition fails when one or more nodes has a `diskPartition` resource configured in the `SiteConfig` CR.
+(link:https://issues.redhat.com/browse/OCPBUGS-9272[*OCPBUGS-9272*])
+
+* In clusters configured with PTP boundary clocks (T-BC) and deployed vDU applications, sometimes messages are not sent from the follower interface of the T-BC on the vDU for periods of up to 40 seconds. The rate or errors in the logs can vary. An example error log is below:
++
+.Example output
+[source,terminal]
+----
+2023-01-15T19:26:33.017221334+00:00 stdout F phc2sys[359186.957]: [ptp4l.0.config] nothing to synchronize
+----
+(link:https://issues.redhat.com/browse/RHELPLAN-145492[*RHELPLAN-145492*])
+
+* RFC2544 performance tests show that the `Max delay` value for a packet to traverse the network is over the minimum threshold. This regression is found in {product-title} 4.13 clusters running the Telco RAN DU profile.
+(link:https://issues.redhat.com/browse/OCPBUGS-13224[*OCPBUGS-13224*])
+
+* When you install a {sno} cluster using {ztp} and configure PTP and bare-metal events with HTTP transport, the `linuxptp-daemon` daemon pod intermittently fails and does not have the required `PersistentVolumeClaim` (`PVC`) CR applied.
```

Comment:
@jzding Can you review this 4.13 known issue please?

Reply:
the `linuxptp-daemon` daemon pod intermittently fails to deploy.